app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Users::SessionsController < Devise::SessionsController
+  before_filter :set_flash, only: [:new]
+
+  private
+
+  def set_flash
+    @flash = Notifier.new flash
```

Comment:
@vlasiak, also, important thing is to name a variable using higher level of abstraction, for example `notification`. That will not couple a view with back-end.